context/context.go

```diff
@@ -73,13 +73,19 @@ func NewClientContext(id int64, conn net.Conn) *ClientContext {
 
 // ServerContext is the runtime context of the server
 type ServerContext struct {
-	RequirePass      string
-	Store            *db.RedisStore
-	Monitors         sync.Map
-	Clients          sync.Map
-	Pause            time.Duration // elapse to pause all clients
-	StartAt          time.Time
-	ListZipThreshold int
+	RequirePass       string
+	Store             *db.RedisStore
+	Monitors          sync.Map
+	Clients           sync.Map
+	LimitersMgr       *db.LimitersMgr
+	Pause             time.Duration // elapse to pause all clients
+	StartAt           time.Time
+	ListZipThreshold  int
+	LimitConnection   bool
+	MaxConnection     int64
```

Comment:
It is too verbose to use two variables here. Maybe we should use MaxConnection with value 0 as unlimited.